feat(ffe-core): nye semantiske farger. [burde merges sammen med ffe-cards og fileupload]

tuva-odegard opened the pull request:
Nye Semantiske farger i core. Gjelder overskrifter og flere skriftvarianter. 
De eksisterende fargene er ikke depricated for å unngå for mye breaking changes. 

Denne commiten påvirker så vidt jeg kan se Cards og FileUpload. Det er fordi den setter skiftfargen til hvit på dark mode, når de komponentene egentlig burde brukt fill. Så det er lurt å gjøre disse før man pusher denne committen, eller med lite mellomrom i alle fall.

dagfrode commented:
Ser at det er vanskelig å jobbe på cards og FileUpload før denne er merget. så tar en frekk en og jobber videre på de i denne branchen så det blir merget samtidig og folker oppdages før merge.

dagfrode committed:
feat(ffe-core): nye semantiske farger.

BREAKING CHANGE: nye farger. Ingen ting å gjøre hvis man ikke har
justert fargene.

dagfrode committed:
feat(ffe-cards-react)!: Semantiske farger

BREAKING CHANGE: Fjerner shadow som paramater siden cards går over til å bruke boarder i stede

dagfrode committed:
feat(ffe-cards)!: Semantiske farger

BREAKING CHANGE: infører semantiske farger

dagfrode committed:
Merge branch 'develop' into semantiske-farger-core

dagfrode committed:
fix(cards-react): fix

dagfrode committed:
Merge branch 'semantiske-farger-core' of github.com:SpareBank1/designsystem into semantiske-farger-core

dagfrode commented:
Ble rot i denne branchen så delt opp i 2 PR
